Mathlib/Analysis/LocallyConvex/Polar.lean

```diff
@@ -5,8 +5,8 @@ Authors: Moritz Doll, Kalle Kytölä
 -/
 import Mathlib.Algebra.EuclideanDomain.Instances
 import Mathlib.Analysis.Normed.Field.Basic
-import Mathlib.LinearAlgebra.SesquilinearForm
 import Mathlib.Topology.Algebra.Module.WeakDual
+import Mathlib.Analysis.NormedSpace.Basic
```

Comment:
Aren't we supposed to put the imports in alphabetical order? I can never remember.